napari_builtins/_tests/conftest.py

```diff
@@ -53,3 +53,8 @@ def _use_builtins(_mock_npe2_pm: PluginManager):
 @pytest.fixture(params=LAYERS)
 def some_layer(request):
     return request.param
+
+
+@pytest.fixture()
+def all_layers():
```

Comment:
It's not really `all` layers is it? Maybe worth renaming it to `basic` or `gui` or something so that in that `test_writer` the usage is more clear?

Reply:
Updated